_data/getstarted-nextsteps.yaml

```diff
@@ -0,0 +1,42 @@
+headline: "Next Steps"
+
+secondarycta: "Want to learn more? Check out the <a href='https://quarkus.io/guides/'>guides</a> to continue your journey."
+
+links:
+- title: "Building Native Executables"
+  description: "Build native executables with GraalVM or Mandrel."
+  link: https://quarkus.io/guides/building-native-image
```

Comment:
Can we use relative links here? For the Surge.sh PR previews

Reply:
Added commit to convert links to relative.